README.md

```diff
@@ -1,83 +1,85 @@
 Riak Dev Cluster for OS X
 =========================
 
-Easily run a 5-node [Riak](http://wiki.basho.com/Riak.html) cluster on
-OS X.
+*NOTE: This project currently installs Riak 2.0.5*
 
+Easily run a 5-node [Riak](http://www.basho.com/riak) cluster on OS X.
+
+The Basho documentation enumerates the [Five Minute Install](http://docs.basho.com/riak/latest/quickstart/). While a straightforward, and useful, exercise to undergo the tutorial is based upon a source build. As a result, in some environments, it can be a time-intensive process.
```

Comment:
I do not quite understand the sentence that begins "While a...". I think you might be able to rewrite the last two sentences as, 
The tutorial is based upon a source build, which can be a time-intensive process in some environments. (It's okay not to have caveats that it is still a good and useful document :) Nothing you say implies otherwise.)